nextjsUIapp/src/app/admin/profile/page.tsx

```diff
@@ -1,44 +1,516 @@
+    ];
+    const [isEditing, setIsEditing] = useState(false);
+    const [activeTab, setActiveTab] = useState("activity");
+
+    const tabs = [
+        { id: "activity", label: "Activity" },
+        { id: "evidenceItems", label: "Evidence Items" },
+        { id: "assertions", label: "Assertions" },
+        { id: "sourceSuggestions", label: "Source Suggestions" },
+    ];
+    const [profileData, setProfileData] = useState({
+        name: session?.user?.name || "",
+        email: session?.user?.email || "",
+        image: session?.user?.image || "",
+        areaOfExpertise: "N/A",
+        role: "Curator",
```

Comment:
Perhaps we should make the `role` field a drop down menu with a predefined set of roles?

Reply:
I chose not to implement the role field as a dropdown to prevent users from changing their roles on their own. The default role, I am thinking of is a `Curator`. Let's discuss in our upcoming meeting.